### ISSUE TITLE
Error posting comment

### ISSUE THREAD

**Issue by crablar** [forem/forem#14502]
This is on our own Forem instance. Why would this be happening?

![Image 8-14-21 at 4 40 PM](https://user-images.githubusercontent.com/1283090/129482175-30f4d1e8-311d-4596-9611-44294ed12dbe.jpg)

**Comment by rhymes** [comment]
@crablar if it's your own instance you probably have access to the server side logs, can you correlate the action of posting an a comment with the error on the other side?

**Comment by fdoxyz** [comment]
I was just able to replicate this in DEV here: https://dev.to/fdoxyz/kubernetes-integration-with-rake-tasks-j0b

<img width="1440" alt="Screen Shot 2021-08-31 at 15 47 11" src="https://user-images.githubusercontent.com/6045239/131580580-8c178ac0-286c-4b8e-90de-ea6afce758ac.png">

I'm going to share this with other team members to try and figure the cause of this bug, I'm not quite sure why this is happening.

**Comment by fdoxyz** [comment]
After some investigating what was happening here was that the user creating the comment was blocked by the author of the Post.

@crablar it's likely one of the following situations was causing the error you encountered:
1. The author of the Post where you tried to comment had blocked you
2. Your account is suspended or "comment suspended"

If you do have access to the admin area of that Forem then you could look up your accounts to ensure you don't have any of the suspended roles. If that's the case then that would mean you likely aren't able to comment on any Post. 

If your account doesn't have any of those roles then I'm not sure if there's a way to tell if you are blocked by that other user unless you are able to communicate with them directly to ask them.

The error message in this situation isn't good, so I'm going to create a separate issue to hopefully improve it in order to provide better feedback when this situation comes up again.

**Comment by cmgorton** [comment]
@fdoxyz since you opened this other issue #14642 do you think this one can be closed or should we keep it open for further investigation?

**Comment by fdoxyz** [comment]
Closing this issue as it was indeed handled in #14642. 

If the error does reappear please feel free to re-open or comment again here.
